package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@figma/widget-typings",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
sorry can we back this out, I can do this separately and then tag the commit with the right version when I publish to npm

Reply:
Done